presto-main/src/test/java/com/facebook/presto/type/TestArrayOperators.java

```diff
@@ -565,6 +565,24 @@ public void testArrayIntersect()
     }
 
     @Test
+    public void testArrayUnion()
```

Comment:
Add tests for `bigint` since you have a custom implementation for that, and make sure to test the case where there are multiple `NULL`s